assets/js/app.js

```diff
@@ -127,6 +127,13 @@ window.addEventListener("lb:clipcopy", (event) => {
   }
 });
 
+// Accessibility handlers
+
+window.addEventListener("lb:live_region:message", (event) => {
+  const message = event.detail.message;
+  event.target.textContent = message;
```

Comment:
We can keep it more generic as we do with `lb:set_value` :)

```suggestion
window.addEventListener("lb:set_text", (event) => {
  event.target.textContent = event.detail.value;
```